packages/compass/src/main/menu.ts

```diff
@@ -472,6 +477,15 @@ class CompassMenu {
       this.refreshMenu();
     });
 
+    preferences.onPreferenceValueChanged('enableDevTools', (enableDevTools: boolean) => {
+      this.refreshMenu();
+      if (enableDevTools) {
+        BrowserWindow.getFocusedWindow()?.webContents.openDevTools();
```

Comment:
Would this open dev tools every time the option is being changed?

Reply:
Yes, I thought it can be handy for feature discoverability. If someone switched it on, they probably want to use it. That being said there is a way to not show it right away.

Reply:
Fair enough, that makes sense :)